paddle/fluid/memory/allocation/allocator_facade.cc

```diff
@@ -145,6 +153,25 @@ class AllocatorFacadePrivate {
     }
   }
 
+  void CheckAllocThreadSafe() const {
+    for (auto& pair : allocators_) {
+      PADDLE_ENFORCE_EQ(pair.second->IsAllocThreadSafe(), true);
+    }
+
+    for (auto& pair : zero_size_allocators_) {
+      PADDLE_ENFORCE_EQ(pair.second->IsAllocThreadSafe(), true);
+    }
+  }
+
+  void WrapCUDARetryAllocator(size_t retry_time) {
+    PADDLE_ENFORCE_GT(retry_time, 0, "Retry time must be larger than 0");
+    for (auto& pair : allocators_) {
+      if (!platform::is_gpu_place(pair.first)) continue;
+
+      pair.second = std::make_shared<RetryAllocator>(pair.second, retry_time);
```

Comment:
Make it shorter without "continue" logic:

if (platform::is_gpu_place(pair.first)) {
    pair.second = std::make_shared<RetryAllocator>(pair.second, retry_time);
}

Reply:
Done.